app/routes/companyInterest/components/CompanyIntrestPageTranslations.tsx

```diff
@@ -0,0 +1,260 @@
+  },
+  company_survey_big_data: {
+    norwegian: 'Big data',
+    english: 'Big data',
+  },
+  company_survey_front_back_end: {
+    norwegian: 'Front- og back-end',
+    english: 'Front- and back-end',
+  },
+  company_survey_iot: {
+    norwegian: 'Internet of Things',
+    english: 'Internet of Things',
+  },
+  company_survey_gamedev: {
+    norwegian: 'Spillutvikling',
+    english: 'Game development',
+  },
+  company_survey_softskills: {
+    norwegian: 'Softskills',
+    english: 'Soft skills',
+  },
+  company_survey_fintech: {
+    norwegian: 'Finansiell teknologi',
+    english: 'Financial technology',
+  },
+};
+
+export const README = {
+  readme: {
+    norwegian: 'Annonse i readme',
+    english: 'Advertisement in readme',
+  },
+  /*
+    collaboration: {
+      norwegian: 'Samarbeid med andre linjeforeninger',
+      english: 'Collaboration with other student organizations',
+    },
+    */
+};
+
+export const COMPANY_TYPES: Record<
+  CompanyInterestCompanyType,
+  { norwegian: string; english: string }
+> = {
+  company_types_small_consultant: {
+    norwegian: 'Liten konsulentbedrift ( < ~50)',
+    english: 'Small consultant company ( < ~50)',
+  },
+  company_types_medium_consultant: {
+    norwegian: 'Medium konsulentbedrift ( < 400)',
+    english: 'Medium consultant company ( < 400)',
+  },
+  company_types_large_consultant: {
+    norwegian: 'Stor konsulentbedrift ( > 400)',
+    english: 'Large consultant company ( > 400)',
+  },
+  company_types_inhouse: { norwegian: 'In-house', english: 'In-house' },
+  company_types_others: { norwegian: 'Annet', english: 'Other' },
+  company_types_start_up: { norwegian: 'Start-up', english: 'Start-up' },
+  company_types_governmental: { norwegian: 'Statlig', english: 'Governmental' },
+};
+
+export const OFFICE_IN_TRONDHEIM = {
+  yes: { norwegian: 'Ja', english: 'Yes' },
+  no: { norwegian: 'Nei', english: 'No' },
+};
+
+export const COLLABORATION_TYPES = {
+  collaboration_omega: {
+    norwegian: 'Samarbeid med Omega linjeforening',
+    english: 'Event in collaboration with Omega',
+  },
+  collaboration_online: {
+    norwegian: 'Samarbeid med Online linjeforening',
+    english: 'Event in collaboration with Online',
+  },
+  collaboration_tihlde: {
+    norwegian: 'Samarbeid med TIHLDE linjeforening',
+    english: 'Event in collaboration with TIHLDE',
+  },
+
+  /*
+      collaboration_anniversary: {
+        english: "Collaboration with Abakus' anniversary committee*",
+        norwegian: 'Samarbeid med Abakus sitt Jubileum*',
+      },
+      collaboration_revue_anniversary: {
+        english: "Collaboration with the revue's anniversary committee*",
+        norwegian: 'Samarbeid med Revyen sitt Jubileum*',
+      },
+      */
+  /*   collaboration_revue: {
+        norwegian: 'Samarbeid med Revyen**',
+        english: 'Collaboration with the revue**',
+      }, */
+};
+export const TARGET_GRADES = {
+  '1': {
+    norwegian: '1. klasse',
+    english: '1st Years',
```

Comment:
Wow it bothers me that `Years` is capitalized and not `klasse`, but I guess that's too nitpicky

Reply:
fixed